pmm_qa/pmm-framework.py

```diff
@@ -30,31 +31,24 @@
     },
 }
 
-def run_ansible_playbook(playbook_filename, env_vars, args):
-    # Install Ansible
-    try:
-        subprocess.run(['sudo', 'yum', 'install', 'ansible', '-y'])
-    except Exception as e:
-        print(f"Error installing Ansible: {e}")
-        exit(1)
 
+def run_ansible_playbook(playbook_filename, env_vars, args):
     # Get Script Dir
     script_path = os.path.abspath(sys.argv[0])
     script_dir = os.path.dirname(script_path)
     playbook_path = script_dir + "/" + playbook_filename
 
-    # Build the commands to execute the playbook
-    command = ["ansible-playbook", f"{playbook_path}", f'-e os_type=linux', f'--connection=local',
-               f'-l localhost', f'-i localhost,']
-
-    if args.verbose:
-        print(f'Options set after considering defaults: {env_vars}')
-
-    try:
-        subprocess.run(command, env=env_vars, check=True)
-    except subprocess.CalledProcessError as e:
-        print(f"Error executing Ansible {command}: {e}")
-        exit(1)
+    print(script_dir)
+    print(playbook_filename)
```

Comment:
I dont think, this is required anymore. If required pls add it as part of if args.verbose: